### ISSUE TITLE
Feature Request: don't show uninstall watcher notification if 0 remnants are found

### ISSUE THREAD

**Issue by jeyjai** [d4rken-org/sdmaid-se#1248]
I like having the notifications for uninstall watcher enabled but would it be possible for you to add a secondary setting to uninstall watcher to only be notified if any remnants are actually found?

**Comment by d4rken** [comment]
This should already be the case, that notifications (that stick around) are only shown if something is found.

Do you maybe mean the progress notifications when the corpsefinder is working? Those are required by Android.

**Comment by jeyjai** [comment]
> This should already be the case, that notifications (that stick around) are only shown if something is found.
> 
> Do you maybe mean the progress notifications when the corpsefinder is working? Those are required by Android.

Every single time i uninstall a app i get a notification from sd maid called uninstall watcher saying it found 0 remnants

**Comment by jeyjai** [comment]
Yes its up there till i sipe it away, or if i uninstall something else, which will replace it with the new app i uninstalled, so there's never more than one.
